source/includes/examples/_cartpole.html.md

```diff
@@ -16,83 +16,73 @@ _A pole is attached by an un-actuated joint to a cart, which moves along a frict
 
 ## Inkling File
 
-###### Schema
-
-```inkling
-schema GameState
-    Float32 position,
-    Float32 velocity,
-    Float32 angle,
-    Float32 rotation
-end
+###### Types
+
+```inkling2
+type GameState {
+    position: number,
+    velocity: number,
+    angle: number,
+    rotation: number
+}
 ```
 
-The schema `GameState` names four records — `position`, `velocity`, `angle`, and `rotation` — and assigns a type to them. This information is input from the simulation.
+The type `GameState` defines four fields - `position`, `velocity`, `angle`, and `rotation` — and assigns a type to each. This information is input from the simulation.
 
-```inkling
-constant Int8 left = 0
-constant Int8 right = 1
-schema Action
-    Int8{left, right} command
-end
+```inkling2
+type Action {
+    command: number<Left = 0, Right = 1>
+}
```

Comment:
Confirming my understanding more than asking for an edit: this doesn't require braces, right?